languages/java/exercises/concept/inheritance/.docs/after.md

```diff
@@ -0,0 +1,55 @@
+## Inheritance
+
+Java supports inheritance as it's core functionality and then to achieve a lot of OOPs principles like abstraction using
+inheritance.
+
+A class can extend another class using `extends` keyword and can inherit from an interface using `implements` keywords.
+
+## Access Modifiers
+
+The access modifiers define rules for the member variables of a class about their access from another classes(or anywhere in
+the code).
+
+There are mainly three access modifiers:
+
+- private
+- public
+- protected
+
+You can read more about them [here][access-modifiers]
+
+## Inheritance vs Composition vs Aggregation
+
+These three concepts are very similar and are often confused.
```

Comment:
I see only two concepts described below this sentence

Reply:
I think I shouldn't mention aggregation at all.